Add pricing information for exposed ports in Jobs (#2550)
huggingface/hub-docs: docs/hub/jobs-pricing.md

>>> hf jobs hardware
```

### Exposed ports

> A Job can expose a port to make it reachable from the outside while it is running. Exposing a port is billed at a flat rate per Job, in addition to the hardware price:

Reviewer: I would add a small example on how to expose a port:

```suggestion
A Job can expose a port to make it reachable from the outside while it is running:

``bash
# Expose a web server running on port 8000
> hf jobs run --expose 8000 python:3.12 python -m http.server 8000
✓ Job started
  id: 6a2aa7cec4f53f9fc5aa4cff
  url: https://huggingface.co/jobs/Wauplin/6a2aa7cec4f53f9fc5aa4cff
Hint: Exposed ports are reachable at (requires an HF token with read access to the job):
  https://6a2aa7cec4f53f9fc5aa4cff--8000.hf.jobs
``

Exposing a port is billed at a flat rate per Job, in addition to the hardware price:

```

Reviewer: (⚠️ code snippet in suggestion above has double-backticks only instead of triple -otherwise GH parses it incorrectly)

> A Job can expose a port to make it reachable from the outside while it is running. Exposing a port is billed at a flat rate per Job, in addition to the hardware price:

Reviewer 2: ```suggestion
A Job can [expose one or more ports](./jobs-configuration#docker-jobs) to make them reachable from the outside while the Job is running. Exposing one or more ports is billed at a flat rate per Job, in addition to the hardware price:
```

Reviewer 3: ^updated to add link to what's shipped in #2551

| **Product**            | **Hourly Price**  |
|----------------------- | ----------------- |

> | Exposed port           | $0.01             |

Reviewer 2: ```suggestion
| Exposed port           | $0.001             |
```

was supposed to be this i think?

Author: You said same price a basic CPU so $0.01/h ($0.000167/minute)

Reviewer 3: ```suggestion
| Exposed ports          | $0.01             |
```

Reviewer 2: no pretty sure i said 1/10th of CPU-basic :)

Reviewer 2: bt no strong opinion
